dcpy/models/lifecycle/ingest.py

```diff
@@ -53,11 +53,14 @@ class Template(BaseModel, extra="forbid", arbitrary_types_allowed=True):
     name: str
     acl: recipes.ValidAclValues
 
+    target_crs: int | None = None
```

Comment:
Bigger picture issue here - we should sit down and think about if we'd like to restructure these template/config objects at all. Wanted to keep this fairly atomic for now and not shuffle things. But that'll be an important step before we wrap up an MVP of the new ingest code

Reply:
Agree. Let's do it next week. 

I do have a question about this `target_crs` attribute. In the Template, it's of int data type; in Config, it's a string. Why is that? 

Note, in the `file.Format`, crs is of a string type like `EPSG:4326`.